WordPress/WordPressTest/Comments/Controllers/FullScreenCommentReplyViewControllerTests.swift

```diff
@@ -60,38 +60,6 @@ class FullScreenCommentReplyViewControllerTests: CoreDataTestCase {
         }
     }
 
-    /// Test if SuggestionsTableView is visible when searchText is provided and it is already opened when text input is collapsed
-    func testSuggestionListVisibleWhenAlreadyVisibleWhenCollapsed() throws {
-        let blog = BlogBuilder(mainContext).build()
-        try mainContext.save()
-        controller = FullScreenCommentReplyViewController(viewModel: viewModel)
-        controller.enableSuggestions(with: blog.dotComID!, prominentSuggestionsIds: [], searchText: "@Ren")
-        controller.content = "Test"
-        load(controller, inWindow: UIWindow())
-
-        let view = controller.navigationController?.view.subviews.first { view in
-            return view is SuggestionsTableView
-        }
-        let suggestionsTableView = try XCTUnwrap(view)
```

Comment:
I spent a good hour trying to debug this but because these tests don't follow best practices for unit testing and are evaluating the UIKit components, I made a call to remove them.